fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -430,6 +430,18 @@ private boolean processDropColumn(DropColumnClause alterClause, OlapTable olapTa
                 throw new DdlException("Column does not exists: " + dropColName);
             }
 
+            // drop bloom filter column
+            Set<String> bfCols = olapTable.getCopiedBfColumns();
+            if (bfCols != null) {
+                Set<String> newBfCols = new HashSet<>();
+                for (String bfCol : bfCols) {
+                    if (!bfCol.equalsIgnoreCase(dropColName)) {
+                        newBfCols.add(bfCol);
+                    }
+                }
+                olapTable.setBloomFilterInfo(newBfCols, olapTable.getBfFpp());
+            }
```

Comment:
may be this can be better
```
Set<String> bfCols = olapTable.getCopiedBfColumns();
if (bfCols != null) {
    Set<String> newBfCols = bfCols.stream()
                                  .filter(bfCol -> !bfCol.equalsIgnoreCase(dropColName))
                                  .collect(Collectors.toSet());
    olapTable.setBloomFilterInfo(newBfCols, olapTable.getBfFpp());
}
```